libraries/AP_HAL_ChibiOS/Util.cpp

```diff
@@ -86,6 +86,26 @@ void *Util::allocate_heap_memory(size_t size)
     return heap;
 }
 
+/*
+  realloc implementation thanks to wolfssl, used by AP_Scripting
+ */
+void *Util::std_realloc(void *addr, size_t size)
+{
+    if (size == 0) {
+       free(addr);
+       return nullptr;
+    }
+    if (addr == nullptr) {
+        return malloc(size);
```

Comment:
Didn't we prefere calloc?

Reply:
malloc in ArduPilot does zero fill, so equivalent to calloc